CHANGELOG.txt

```diff
@@ -1,3 +1,6 @@
+v2.0.6 (11/10/2016)
+* Changed Step intents to PlayerSeek for easier grouping, functionality unchanged.
+
```

Comment:
I don't think we need to release a new version for a non-functional change.  It probably doesn't even need a mention in the changelog, honestly.